### PR TITLE
Add: new_relic monitoring plugin to our site

### PR DESCRIPTION
**Pull request by jakshi**

### CODE REVIEW

File: src/templates/haml/layouts/default/head.html.haml [hunk 1]
```diff
@@ -26,6 +26,8 @@
 
   %title= stash('website_name') . (stash('title') ? " : ".stash('title'): '')
 
+  %script{:type => "text/javascript", :src => url_for('/js/newrelic.js')}
```
Comment (raunakkathuria) [inline src/templates/haml/layouts/default/head.html.haml]:
```
%script{:type => "text/javascript"}
   != "code here"
```

File: src/templates/haml/layouts/default/head.html.haml [hunk 2]
```diff
@@ -26,6 +26,11 @@
...
...),"undefined"==typeof n&&(n=t?t.length:0);for(var r=-1,o=n-e||0,i=Array(0>o?0:o);++r<o;)i[r]=t[e+r];return i}e.exports=n},{}],24:[function(t,e){function n(t){return!(t&&"function"==typeof t&&t.apply&&!t[i])}var r=t("ee"),o=t(1),i="nr@wrapper",a=Object.prototype.hasOwnProperty;e.exports=function(t){function e(t,e,r,a){function nrWrapper(){var n,i,s,f;try{i=this,n=o(arguments),s=r&&r(n,i)||{}}catch(d){u([d,"",[n,i,a],s])}c(e+"start",[n,i,a],s);try{return f=t.apply(i,n)}catch(p){throw c(e+"err",[n,i,p],s),p}finally{c(e+"end",[n,i,f],s)}}return n(t)?t:(e||(e=""),nrWrapper[i]=!0,f(t,nrWrapper),nrWrapper)}function s(t,r,o,i){o||(o="");var a,s,c,f="-"===o.charAt(0);for(c=0;c<r.length;c++)s=r[c],a=t[s],n(a)||(t[s]=e(a,f?s+o:o,i,s))}function c(e,n,r){try{t.emit(e,n,r)}catch(o){u([o,e,n,r])}}function f(t,e){if(Object.defineProperty&&Object.keys)try{var n=Object.keys(t);return n.forEach(function(n){Object.defineProperty(e,n,{get:function(){return t[n]},set:function(e){return t[n]=e,e}})}),e}catch(r){u([r])}for(var o in t)a.call(t,o)&&(e[o]=t[o]);return e}function u(e){try{t.emit("internal-error",e)}catch(n){}}return t||(t=r),e.inPlace=s,e.flag=i,e}},{1:23,ee:"QJf3ax"}]},{},["G9z0Bl",4,11,5]);
```
Comment (borisyankov) [inline src/templates/haml/layouts/default/head.html.haml]: Put that into a separate file, then embed.

Comment (borisyankov) [inline src/templates/haml/layouts/default/head.html.haml]: Is it needed to load before everything else?
If not, better be in an async script tag.

File: config.json [hunk 3]
```diff
@@ -2,7 +2,7 @@
     "website_name": "Binary.com",
     "display_languages": "EN,ID,RU,ES,FR,PT,DE,ZH_CN,PL,AR,ZH_TW,VI,IT",
     "static": {
-        "url": "https://static.binary.com/",
+        "url": "https://static.binaryqa08.com/",
```
Comment (borisyankov) [inline config.json]: Don't commit that ;)

**Pull request closed without merging**